### PR TITLE
common-arguments, style-guide: add Korean translation

### PR DESCRIPTION
**Pull request by KKWANH**
- [x] The page(s) are in the correct platform directories: `common`, `linux`, `osx`, `windows`, `sunos`, `android`, etc.
- [x] The page(s) have at most 8 examples.
- [x] The page description(s) have links to documentation or a homepage.
- [x] The page(s) follow the [content guidelines](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#guidelines).
- [x] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#commit-message).

### CODE REVIEW

**Comment by kbdharun** [comment]
Welcome to tldr. I had requested reviews from others. regarding this PR. I hope it will be merged soon. 

If possible can you add Korean Translation to [common-arguments](https://github.com/tldr-pages/tldr/blob/main/contributing-guides/translation-templates/common-arguments.md) in a new PR will be a helpful reference for reviewers and other translators in future?

> Thanks in advance

**Comment by KKWANH** [comment]
> Welcome to tldr. I had requested reviews from others. regarding this PR. I hope it will be merged soon.
> 
> If possible can you add Korean Translation to [common-arguments](https://github.com/tldr-pages/tldr/blob/main/contributing-guides/translation-templates/common-arguments.md) in a new PR will be a helpful reference for reviewers and other translators in future?
> 
> > Thanks in advance

I commited it just now.

**Comment by kbdharun** [comment]
> I committed it just now.

Thanks

File: contributing-guides/style-guide.ko.md [hunk 1]
```diff
@@ -0,0 +1,165 @@
...
+## 레이아웃
+
+각 페이지의 기본 포맷은 다음 템플릿과 일치해야 하며,<br>
+다음과 최대 8개의 명령어 예제를 포함해야 합니다:
+```md
+# 명령어 이름
+
+> 짧고 간결한 설명
+> 보통 1줄, 필요한 2줄 까지 허용됨
+> <https://example.com/command_name/help/page>에서 더 많은 정보를 찾을 수 있습니다.
```
Comment (blueskyson) [inline contributing-guides/style-guide.ko.md:14]: - Put a periods in description sentences.
- Conventionally, we put spaces between full-width characters and Latin characters.
- We have a Korean translation of "more information" in [set-more-info-link.py](https://github.com/tldr-pages/tldr/blob/main/scripts/set-more-info-link.py).
```suggestion
> 짧고 간결한 설명.
> 보통 1 줄, 필요한 2 줄 까지 허용됨.
> 더 많은 정보: <https://example.com/command_name/help/page>.
```

File: contributing-guides/style-guide.ko.md [hunk 2]
```diff
@@ -0,0 +1,165 @@
...
+`command_name options`
+
+...
+```
+
+예시:
+```md
+# 명령어 이름
+```
+# krita
```
Comment (blueskyson) [inline contributing-guides/style-guide.ko.md:31]:
```suggestion
예시:

```md
# krita
```

File: contributing-guides/style-guide.ko.md [hunk 3]
```diff
@@ -0,0 +1,165 @@
...
+
+> krita는 디지털 아티스트를 위해 설계된 스케치/페인팅 프로그램입니다.
+> `gimp` 페이지도 참조하세요.
+> 더 많은 정보는 <https://docs.krita.org/en/reference_manual/linux_command_line.html>를 참조하세요.
```
Comment (blueskyson) [inline contributing-guides/style-guide.ko.md:35]:
```suggestion
> krita 는 디지털 아티스트를 위해 설계된 스케치/페인팅 프로그램입니다.
> `gimp` 페이지도 참조하세요.
> 더 많은 정보: <https://docs.krita.org/en/reference_manual/linux_command_line.html>.
```

File: contributing-guides/style-guide.ko.md [hunk 4]
```diff
@@ -0,0 +1,165 @@
...
+
+- 로딩 화면 없이 시작:
+
+`krita --nosplash`
+
+- 특정 파일 열기:
+
+`krita {{path/to/image1 path/to/image2 ...}}`
+
+- 특정 workspace(`Animation`)에서 시작:
```
Comment (blueskyson) [inline contributing-guides/style-guide.ko.md:49]: Conventionally, we put spaces between full-width characters and half-width punctuations.
```suggestion
- 특정 workspace (`Animation`) 에서 시작:
```

**Review by blueskyson — COMMENTED** [review]
Thanks for contribution. Currently we don't have Korean reviewer, so I make some suggestions based on the Chinese specific rules of tldr. Please let me know if there's any wrong Korean copywriting suggestion.

File: contributing-guides/style-guide.ko.md (hunk 2 above)
Comment (blueskyson) [reply contributing-guides/style-guide.ko.md:31]: There is one extra "```" in line 30.

File: contributing-guides/style-guide.ko.md (hunk 1 above)
Comment (KKWANH) [reply contributing-guides/style-guide.ko.md:14]: In Korean, we don't need to separate number and the letter with space. 1줄, 2줄 is no problem.
And "more information" part, I will emend my pull request later.
Thank you!

**Comment by blueskyson** [comment]
The "krita" markdown block should be like the right side.
Screenshot from 2022-10-09 18-09-19

**Comment by KKWANH** [comment]
Sorry, it was my mistake, I am not familiar with git...

File: contributing-guides/style-guide.ko.md [hunk 5]
```diff
@@ -0,0 +1,166 @@
...
+
+## Imperative Mood
+
+예시 설명형은 명령법으로 표현되어야 합니다.
+예를 들자면 `List all files.`를 `Listing all files`, `File listing` 등 입니다.
+이것은 특별한 경우를 제외하고 기본적으로 **모든 번역에 적용**됩니다.
+
+## Serial Comma
+
+3개 이상의 항목 목록을 선언할 때, Oxford 쉼표라고도 부르는 [연속 쉼표(https://en.wikipedia.org/wiki/Serial_comma)]를 사용합니다.
```
Comment (blueskyson) [inline contributing-guides/style-guide.ko.md:151]:
```suggestion
3개 이상의 항목 목록을 선언할 때, Oxford 쉼표라고도 부르는 [연속 쉼표](https://en.wikipedia.org/wiki/Serial_comma)를 사용합니다.
```

Comment (KKWANH) [reply contributing-guides/style-guide.ko.md]: thanks!

**Review by blueskyson — APPROVED** [review]

**Review by blueskyson — COMMENTED** [review]
Thanks for contribution. Please work on another branch instead of editing main branch directly next time.

**Review by kbdharun — APPROVED** [review]
LGTM, Thanks for this massive PR ❤️.

> Checked the files using Google Translate

**Comment by kbdharun** [comment]
@KKWANH , Now that the PR is [merged](https://github.com/tldr-pages/tldr/blob/main/contributing-guides/style-guide.ko.md). I have some suggestions for style guide it would be better if the paths in naming can be translated too. Also, The more information text in more information links can be translated to [더 많은 정보](https://github.com/tldr-pages/tldr/blob/main/scripts/set-more-info-link.py). If possible can you create a new PR for it?

> Thanks in advance.

**Pull request merged**